app-admin/qtpass/qtpass-1.2.1.ebuild

```diff
@@ -25,25 +25,48 @@ RDEPEND="app-admin/pass
 	dev-qt/qtwidgets:5
 	net-misc/x11-ssh-askpass"
 DEPEND="${RDEPEND}
-	dev-qt/linguist-tools:5"
+	dev-qt/linguist-tools:5
+	dev-qt/qtsvg:5
+	test? ( dev-qt/qttest:5 )"
+
+S="${WORKDIR}/${MY_P}"
+
+DOCS=( CHANGELOG.md CONTRIBUTING.md FAQ.md README.md  )
 
 src_prepare() {
 	default
 
-	sed -i 's/SUBDIRS += src tests main/SUBDIRS += src main/' "${S}"/qtpass.pro || die
-	sed -i '/main\.depends = tests/d' "${S}"/qtpass.pro || die
+	sed -i '/target.path = /s/$$\[QT_INSTALL_TESTS\]/$$PREFIX\$$[QT_INSTALL_TESTS\]/' \
+		tests/tests.pri || die "sed for tests.pri failed"
+	if ! use test ; then
+		sed -i \
+			-e '/SUBDIRS += src /s/tests //' \
+			-e '/main.depends = /s/tests/src/' \
+				qtpass.pro || die "sed for qtpass.pro failed"
+	fi
+
+	local lr=$(qt5_get_bindir)/lrelease
```

Comment:
Please don't run lrelease inside `src_prepare`. It is fine if you move it to `src_compile` instead.

Reply:
Moved to src_compile section.